decryptor/mysql/response_proxy.go

```diff
@@ -342,7 +342,8 @@ func (handler *Handler) ProxyClientConnection(errCh chan<- error) {
 			handler.setQueryHandler(handler.QueryResponseHandler)
 			break
 		case CommandStatementClose, CommandStatementSendLongData, CommandStatementReset:
-			fallthrough
+			clientLog.Debugln("Close|SendLongData|Reset command")
+			break
```

Comment:
```suggestion
```

Reply:
yeah, golang doesn't require break

Reply:
really?
my main concern was about the reason of changing logic – from "fallthrough" to "break"